Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
+	public function do_nothing(): void {
```

Comment:
:no_entry_sign: You must use "/**" style comments for a class comment

<sub>hash: 3fda944041338d326807f99503c2d731</sub>